packages/sdk/contractkit/src/wrappers/FederatedAttestations.ts

```diff
@@ -1,8 +1,171 @@
 import { FederatedAttestations } from '../generated/FederatedAttestations'
-import { BaseWrapper } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend } from './BaseWrapper'
+import { CeloTransactionObject } from '@celo/connect'
 
-// TODO ASv2 -- add params or delete config if there are none
-// & delete other commented-out usages of this Config in CK
-// export interface FederatedAttestationsConfig {}
+type Address = string
+type Identifier = string | number[]
 
-export class FederatedAttestationsWrapper extends BaseWrapper<FederatedAttestations> {}
+export class FederatedAttestationsWrapper extends BaseWrapper<FederatedAttestations> {
+  /**
+   * @notice Returns identifiers mapped to `account` by signers of `trustedIssuers`
+   * @param account Address of the account
+   * @param trustedIssuers Array of n issuers whose identifier mappings will be used
+   * @return countsPerIssuer Array of number of identifiers returned per issuer
+   * @return identifiers Array (length == sum([0])) of identifiers
+   * @dev Adds identifier info to the arrays in order of provided trustedIssuers
+   * @dev Expectation that only one attestation exists per (identifier, issuer, account)
+   */
+  lookupIdentifiers: (
+    account: Address,
+    trustedIssuers: Address[]
+  ) => Promise<{
+    countsPerIssuer: string[]
+    identifiers: string[]
```

Comment:
why is number sometimes an identifier and sometime only string?

Reply:
Good question - this is just the return type of this function. But for all parameters, Identifiers are string | number[]